website/pages/api-docs/secret/openldap/index.mdx

```diff
@@ -0,0 +1,217 @@
+---
+layout: api
+page_title: OpenLDAP - Secrets Engines - HTTP API
+sidebar_title: OpenLDAP
+description: This is the API documentation for the Vault OpenLDAP secrets engine.
+---
+
+# OpenLDAP Secrets Engine (API)
+
+This is the API documentation for the Vault OpenLDAP secrets engine. For general
+information about the usage and operation of
+the OpenLDAP secrets engine, please see [these docs](/docs/secrets/openldap).
+
+This documentation assumes the OpenLDAP secrets engine is enabled at the `/openldap` path
+in Vault. Since it is possible to mount secrets engines at any path, please
+update your API calls accordingly.
+
+## Configuration Management 
+
+| Method   | Path               |
+| :-----   | :----------------- |
+| `POST`   | `/openldap/config` |
+| `GET`    | `/openldap/config` |
+| `DELETE` | `/openldap/config` |
+
+This endpoint configures the OpenLDAP secret engine to managed user entries.
+
+Note: the OpenLDAP entry used by `config` should have the neccessary privileges 
+to search and change entry passwords in OpenLDAP.
+
+### Parameters
+
+- `binddn` (string, required) - Distinguished name (DN) of object to bind for managing user entries. Example: `cn=vault,ou=Users,dc=hashicorp,dc=com`
```

Comment:
Are these copy pasta'd from other LDAP docs? If so, I won't suggest any formatting changes. If they're not copied, I do notice we've been doing fields differently lately, like this:

```
- `public_key` `(string: <required>)` – Lorum ipsum.
```

Reply:
These are actually from the AD doc, but I can look into the deltas!

Reply:
Updated!